workflow/templateresolution/context_test.go

```diff
@@ -72,6 +72,22 @@ spec:
       image: docker/whalesay
       command: [cowsay]
       args: ["{{inputs.parameters.message}}"]
+  - name: whalesay-with-passthrough-arguments
+    template: whalesay-with-passthrough-arguments-inner
+    inputs:
+      parameters:
+      - passthroughRegexp: ".*"
```

Comment:
Should it be `regexp` instead of `passthroughRegexp`?

Reply:
I think we should add validation to avoid `Inputs` having `PassthroughRegexp` and `Arguments` having `Regexp`.

Reply:
Agreed, will commit.